src/main/java/com/github/maracas/roseau/diff/changes/BreakingChangeKind.java

```diff
@@ -29,7 +29,175 @@ public enum BreakingChangeKind {
 	TYPE_FORMAL_TYPE_PARAMETERS_ADDED(MUTATION),
 	TYPE_FORMAL_TYPE_PARAMETERS_REMOVED(MUTATION),
 	TYPE_FORMAL_TYPE_PARAMETERS_CHANGED(MUTATION),
-
+	/**
+	 * A breaking change where a method has been removed or its accessibility has
+	 * been reduced.
+	 * <p>
+	 * This occurs when a method that was present in the old version of a library is
+	 * either:
+	 * <p>
+	 * - No longer present in the new version.
+	 * <p>
+	 * - Its visibility has been reduced, making it inaccessible from some or all of
+	 * the previously allowed access points.
+	 * <p>
+	 * - It has been overloaded or its parameters have been changed, effectively
+	 * removing the original method.
+	 * <p>
+	 * - A class or interface no longer extends or implements the parent class or
+	 * interface that contains the method.
+	 *
+	 * <p>
```

Comment:
I think this is quite long so maybe we do not include the examples here but instead in a Wiki no?

Reply:
And maybe we add a link to the wiki page?